README.md

```diff
@@ -350,12 +350,12 @@ If your custom field resolves an object you can expose that to CraftQL as well.
 Event::on(\craft\base\Field::class, 'craftQlGetFieldSchema', function ($event) {
   $field = $event->sender;
 
-  $object = $event->schema->createObjectType('MapPoint')
+  $object = $event->schema->createObjectType(ucfirst($field->handle) . 'MapPoint')
```

Comment:
I don't think we need the `$field->handle` on this.

```suggestion
  $object = $event->schema->createObjectType('MapPoint')
```

If we include the `$field->handle` then ever field is going to need `...on FieldHandleMapPoint` even though every map point has the same `lat`, `lng`, and `zoom` fields.